app/routes/detail.ts

```diff
@@ -110,11 +101,16 @@ export default class DetailRoute extends Route {
         size: 4,
       },
       municipality: agendaItem.session?.get('municipality') || undefined,
+      include: agendaItemIncludes,
       filter: {
-        session: {
+        sessions: {
           'governing-body': {
-            'administrative-unit': {
-              name: agendaItem.session?.get('municipality') || undefined,
+            'is-time-specialization-of': {
+              'administrative-unit': {
+                location: {
+                  label: agendaItem.session?.get('municipality') || undefined,
```

Comment:
`.get` shouldn't be needed if the session getter returns the record instead of the proxy.